gateways/adyen/request_builder_test.go

```diff
@@ -16,6 +16,9 @@ func TestBuildAuthRequest(t *testing.T) {
 	base := sleet_testing.BaseAuthorizationRequest()
 	requestWithLevel3Data := sleet_testing.BaseAuthorizationRequest()
 	requestWithLevel3Data.Level3Data = sleet_testing.BaseLevel3Data()
+	requestWithLevel3ItemDiscount := sleet_testing.BaseAuthorizationRequest()
+	requestWithLevel3ItemDiscount.Level3Data = sleet_testing.BaseLevel3Data()
+	requestWithLevel3ItemDiscount.Level3Data.LineItems[0].ItemDiscountAmount.Amount = 100
```

Comment:
where did the `100` come from?

Reply:
Just some random number haha

Reply:
oh i didn't realize this was a test whoops